go/felix/intdataplane/ipip_mgr_test.go

```diff
@@ -0,0 +1,310 @@
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package intdataplane
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"errors"
+	"fmt"
+	log "github.com/Sirupsen/logrus"
+	"github.com/projectcalico/felix/go/felix/ipsets"
+	. "github.com/vishvananda/netlink"
```

Comment:
I don't mind making that change.  It does make some of the code a bit verbose/need line wrapping.